SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -480,10 +525,27 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 		}
 
 		TOptional<Trace_SpanId> SpanId;
-		if (EventTracer != nullptr && EventTracer->IsEnabled())
+		if (EventTracer != nullptr && CauseSpanId.IsSet()
+			&& RepChanges->RepChanged.Num() > 0) // Only need to add these if they are actively being traced
```

Comment:
Is this an optimisation over last time?

Reply:
Possibly but the real reason is because we trace individual properties now instead of the entire update. So RepChanges is used - if no properties changed there is nothing to trace!